### DISCUSSION TITLE
Is it possible to install a universal build of a package? Both for x86_64 and arm64

### DISCUSSION THREAD

**Discussion by dortamiguel** [Homebrew/discussions#1666] [Getting started]
Hi, I will like to be able to get a universal build for sdl2 and sdl2_mixer.

Right now I can only get a library that will run on an arm64 mac, is there an option to tell brew to build sdl2 as a universal package?

Thank you

**Comment by carlocab** [comment]
Unfortunately not. See https://github.com/Homebrew/discussions/discussions/1609, https://github.com/Homebrew/discussions/discussions/1225.

**Reply by dortamiguel** [reply]
Not even a flag I can pass to build this locally?

**Reply by dortamiguel** [reply]
I see there is a tool called `lipo` maybe I could use this merge both sdl2 x86_64 and arm64 dlyb into one, right?

Is it possible to install both architectures with brew so I can later use this tool?

**Reply by carlocab** [reply]
> I see there is a tool called `lipo` maybe I could use this merge both sdl2 x86_64 and arm64 dlyb into one, right?

In principle, yes. Note that these are built with different prefixes, so you'll need to use `install_name_tool` to change dylib IDs and install names. After that, there may still be remaining references to the build prefix that could lead to problems.

> Is it possible to install both architectures with brew so I can later use this tool?

Not really, no.

**Reply by dortamiguel** [reply]
I see, it could be done but too much work. I think Ill just wait for SDL_Mixer to have an official universal release
